specification/schemas/responses/MessageResponse.yaml

```diff
@@ -56,6 +56,12 @@ properties:
                   type: string
                   description: If there is extra information associated with the status of this channel, it is provided here.
                   example: ""
+                supplierStatus:
+                  description: |-
+                    The current status of this message within the channel at the time this response was generated.
+      
+                    For more information please check our documentation on message & channel statuses above.
+                  $ref: ../enums/SupplierStatus.yaml
```

Comment:
Will this always exist in each channel regardless of whether it's an NHSAPP channel?

Reply:
After the next epic, yes ;)